cea/optimization/master/cost_model.py

```diff
@@ -149,8 +150,9 @@ def addCosts(indCombi, buildList, locator, dicoSupply, QUncoveredDesign, QUncove
         # Boiler Base
         if dicoSupply.Boiler_on == 1:
             Q_design = dicoSupply.Boiler_Q_max
-            
-            fNameSlavePP = dicoSupply.configKey + "PPActivationPattern.csv"
+
+            fNameSlavePP = locator.get_optimization_slave_PPActivationPattern(dicoSupply.configKey)
+            # fNameSlavePP = dicoSupply.configKey + "PPActivationPattern.csv"
```

Comment:
delete lines you're replacing. don't comment them out! this just creates a mess nobody will ever be able to understand! (we use git to track what the line used to be)